apps/galaxy/src/services/utils/axios.ts

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
```

Comment:
If this is a utility file, then I feel like having it in `@src/utils` would make more sense. But if it's purely for services, then I reckon `@src/services/axios.ts` would suffice.